flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendLoadingTest.java

```diff
@@ -227,7 +228,7 @@ public void testLoadFileSystemStateBackend() throws Exception {
 		final String savepointDir = new Path(tmp.newFolder().toURI()).toString();
 		final Path expectedCheckpointsPath = new Path(checkpointDir);
 		final Path expectedSavepointsPath = new Path(savepointDir);
-		final int threshold = 1000000;
+		final MemorySize threshold = MemorySize.parse("900kb");
```

Comment:
Do you think we need to add a test case to verify the default value of `state.backend.fs.memory-threshold`(do not set this key, and verify the result of `getMinFileSizeThreshold()` is the expected value)

Reply:
I think current [test_get_min_file_size_threshold](https://github.com/apache/flink/blob/e92b2bf19bdf03ad3bae906dc5fa3781aeddb3ee/flink-python/pyflink/datastream/tests/test_state_backend.py#L100) should already verify it.